Jenkinsfile

```diff
@@ -1,6 +1,11 @@
 library 'ableton-utils@0.13'
 library 'groovylint@0.6'
 
+import groovy.transform.Field
+
+
+@Field static final String CREDENTIAL_ID = 'jenkins-node-scanner-password'
```

Comment:
Would it make more sense to just use this string directly? Importing `Field` and creating a global seems like a lot of work just to save breaking a line (which is what I assume is going on on line 48—correct me if I'm wrong).

Reply:
Yes, the goal was to avoid a linebreak, but this also mirrors how we handle credentials in other projects.

Reply:
Fixed in 2297e81.